commit c067439d9fc04046b9e8996b4b07f822c6873403
Author: Romain Beauxis <toots@rastageeks.org>
Date:   Sun Jan 2 10:19:50 2022 -0600

    Bump windows ffmpeg packages, install tsdl-image with fix in docker images, fix type for last_metadata.

diff --git a/src/lang/lang.ml b/src/lang/lang.ml
index 2dc4e28e..490ef04a 100644
--- a/src/lang/lang.ml
+++ b/src/lang/lang.ml
@@ -608,7 +608,7 @@ let source_methods =
        initialized.",
       fun s -> val_fun [] (fun _ -> bool s#is_ready) );
     ( "last_metadata",
-      ([], fun_t [(false, "", fun_t [] (nullable_t metadata_t))] unit_t),
+      ([], fun_t [] (nullable_t metadata_t)),
       "Return the last metadata from the source.",
       fun s ->
         val_fun [] (fun _ ->
